requirements.txt

```diff
@@ -1,4 +1,5 @@
 cirq-unstable==0.8.0.dev20200318154950
+scikit-learn
```

Comment:
can you delmit this requirement with spaces and a comment

```python

# optimize
scikit-learn
```

Reply:
Done.